dev/plots_api_strawman.py

```diff
@@ -0,0 +1,242 @@
+#!/usr/bin/python
+
+# ======================================
+# Plots for Observation Planning Toolbox
+# ======================================
+
+# This is a strawman API for plotting with Astroplan.
+
+# Must first construct an Observer object using astropy.coordinates.
+
+from astroplan import Observer
+from astropy.coordinates import EarthLocation
+import astropy.units as u
+from pytz import timezone
+
+my_location = EarthLocation.from_geodetic(
```

Comment:
Very minor point, but I would think it might make more sense to include this below in the ``Observer`` call because it's obviously associated with Subaru.  Or if you really like it separate, I would change ``my_location`` to something like ``subaru_location`` to make it more obvious it's really meant for use there.

Also, to make it slightly more explicit you can do ``EarthLocation(lon='-155d28m48.900s', lat='+19d49m42.600s',height=4163*u.m)

(These are all mostly personal opinion, though, so if you genuinely think it's better this way  that's ok too!)